package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "mongo-demo",
+  "version": "1.0.0",
+  "main": "server.js",
+  "scripts": {
+    "test": "set DEBUG=list* && mocha",
+    "start": "set DEBUG=list* && node server.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "bluebird": "^3.4.7",
+    "body-parser": "^1.15.2",
+    "cors": "^2.8.1",
+    "debug": "^2.6.0",
+    "express": "^4.14.0",
+    "http-errors": "^1.5.1",
+    "mongoose": "^4.7.5",
+    "morgan": "^1.7.0"
+  },
+  "directories": {
+    "test": "test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bhavyaab/13-mongo_and_express_api.git"
+  },
+  "bugs": {
+    "url": "https://github.com/bhavyaab/13-mongo_and_express_api/issues"
+  },
+  "homepage": "https://github.com/bhavyaab/13-mongo_and_express_api#readme",
+  "description": "",
+  "devDependencies": {
```

Comment:
Most of these things should only be installed in your dependencies. You need to double check all your dependencies and devDependencies and think about which ones you need when. Remember the things in your devDependencies should only be needed during the development of your app.